components/reusable-components/footer/Footer.tsx

```diff
@@ -107,6 +125,13 @@ const Footer: React.FC = () => {
           <div className={`title-l ${styles.copyrightSection}`}>
             <p>&copy; 2024 Copyright by LearnHub. All rights reserved.</p>
           </div>
+
+          <Turnstile
+            sitekey={process.env.NEXT_PUBLIC_SITE_KEY || ''}
+            onVerify={(token) => setTurnstileToken(token)}
+            theme="light"
```

Comment:
Even though the captcha is invisible, it should be matching the theme of the website.